src/gui/MainWindow.cpp

```diff
@@ -64,6 +64,24 @@
 
 #include "lmmsversion.h"
 
+#ifdef LMMS_BUILD_LINUX
+#include <dlfcn.h>
+//This function is slightly modified from a solution to a KDE bug
+//Found at https://bugs.kde.org/show_bug.cgi?id=337491#c21
+void DisableSystemAccel(QWidget *what)
+{
+	void *d = dlopen("libKF5WidgetsAddons.so", RTLD_LAZY);
```

Comment:
Nitpicking, but... 
1. We should prefer something more descriptive such as `void *handle = dlopen(...)`.
2. This function name should start with a lowercase letter.
3. I find the name to be agnostic to hardware acceleration, please give it a name which is less agnostic, e.g. `disableAutoMnemonics` or something about "[Keys](http://doc.qt.io/qt-4.8/accelerators.html)".
4. We have a lot of workaround code we've borrowed and a simple link should be enough.  e.g. "Disable auto-keyboard accelerators for KDE per https://bugs.kde.org/show_bug.cgi?id=337491"

Reply:
The function name is referring to keyboard accelerators, but I agree, the naming isn't great.